src/components/CreateGoals.js

```diff
@@ -15,23 +15,28 @@ function CreateApp() {
         Open Sauced is a tool to help track your open source contributions. You can get started by creating a goal
         workspace below.
       </p>
-      <p>A public repository name "open-sauced-goals" will be created on your GitHub account to store.</p>
+      <p>A public repository named "open-sauced-goals" will be created on your GitHub account to store data about your goals.</p>
       <small>
         <em>You own all your data saved while saucin.</em>
       </small>
     </React.Fragment>
   );
 }
 
-function InstallApp() {
+function InstallApp({user}) {
+  const repoUrl = "https://github.com/" + user.login + "/open-sauced-goals";
```

Comment:
Maybe we could use [Template literals](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals) for this string

Reply:
Clearly showing my age here LOL.